pkg/operator/ceph/csi/version.go

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright 2020 The Rook Authors. All rights reserved.
```

Comment:
can you please add unit tests for this file?

Reply:
Done